lms/views/lti/basic_launch.py

```diff
@@ -128,6 +128,9 @@ def configure_assignment(self):
         if group_set := self.request.parsed_params.get("group_set"):
             extra["group_set_id"] = group_set
 
+        # Make any product-specific actions after configuring the assignment
+        self.request.product.plugin.misc.post_configure_assignment(self.request)
```

Comment:
Should this also be in the equivalent `deep_linked` method?

Reply:
Added a note about deep linking in the plugin.